sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/implementation/EncryptionProcessor.java

```diff
@@ -221,15 +219,6 @@ public CosmosEncryptionAsyncClient getEncryptionCosmosClient() {
         return encryptionCosmosClient;
     }
 
-    /**
-     * Gets the provider that allows interaction with the master keys.
-     *
-     * @return encryptionKeyWrapProvider
-     */
-    public EncryptionKeyWrapProvider getEncryptionKeyWrapProvider() {
-        return encryptionKeyWrapProvider;
-    }
-
     public EncryptionSettings getEncryptionSettings() {
```

Comment:
missing java doc?

Reply:
These are internal class, and lots of places we are not having doc both for core and encryption lib, good to have docs . So will  track it separately

Reply:
oh yea make sense, missed this is an internal class